code/weapons/Pistol.cs

```diff
@@ -29,6 +29,11 @@ public override bool CanPrimaryAttack()
 
         public override void AttackPrimary()
         {
+            if (TimeSinceDeployed < 0.4f)
```

Comment:
Is there any way to use the `deploy` animation instead of just hard coding values?

Reply:
Yea, for sure thought the same before searching these values, but didn't found the right method / var if there was one. Gonna try to add a var to Weapons.cs so we don't need to check it in every weapon manually (maybe found a better way).